packages/admin-next/dashboard/src/routes/orders/order-detail/components/order-summary-section/order-summary-section.tsx

```diff
@@ -0,0 +1,239 @@
+  order: Order
+}
+
+export const OrderSummarySection = ({ order }: OrderSummarySectionProps) => {
+  return (
+    <Container className="divide-y divide-dashed p-0">
+      <Header order={order} />
+      <ItemBreakdown order={order} />
+      <CostBreakdown order={order} />
+      <Total order={order} />
+    </Container>
+  )
+}
+
+const Header = ({ order }: { order: Order }) => {
+  const { t } = useTranslation()
+
+  return (
+    <div className="flex items-center justify-between px-6 py-4">
+      <Heading level="h2">{t("fields.summary")}</Heading>
+      <ActionMenu
+        groups={[
+          {
+            actions: [
+              {
+                label: t("orders.summary.editItems"),
+                to: "#", // TODO: Open modal to edit items
+                icon: <PencilSquare />,
+              },
+              {
+                label: t("orders.summary.allocateItems"),
+                to: "#", // TODO: Open modal to allocate items
+                icon: <Buildings />,
+              },
+            ],
+          },
+        ]}
+      />
+    </div>
+  )
+}
+
+const Item = ({
+  item,
+  currencyCode,
+  reservation,
+}: {
+  item: LineItem
+  currencyCode: string
+  reservation?: ReservationItemDTO | null
+}) => {
+  const { t } = useTranslation()
+
+  return (
+    <div
+      key={item.id}
+      className="text-ui-fg-subtle grid grid-cols-2 items-start gap-x-4 px-6 py-4"
+    >
+      <div className="flex items-start gap-x-4">
+        <Thumbnail src={item.thumbnail} />
+        <div>
+          <Text
+            size="small"
+            leading="compact"
+            weight="plus"
+            className="text-ui-fg-base"
+          >
+            {item.title}
+          </Text>
+          {item.variant.sku && (
+            <div className="flex items-center gap-x-1">
+              <Text size="small">{item.variant.sku}</Text>
+              <Copy content={item.variant.sku} className="text-ui-fg-muted" />
+            </div>
+          )}
+          <Text size="small">
+            {item.variant.options.map((o) => o.value).join(" · ")}
+          </Text>
+        </div>
+      </div>
+      <div className="grid grid-cols-3 items-center gap-x-4">
+        <div className="flex items-center justify-end gap-x-4">
+          <Text size="small">
+            {getLocaleAmount(item.unit_price, currencyCode)}
+          </Text>
+        </div>
+        <div className="flex items-center gap-x-2">
+          <div className="w-fit min-w-[27px]">
+            <Text>
+              <span className="tabular-nums">{item.quantity}</span>x
+            </Text>
+          </div>
+          <div className="overflow-visible">
+            <StatusBadge
+              color={reservation ? "green" : "orange"}
+              className="text-nowrap"
+            >
+              {reservation
+                ? t("orders.reservations.allocatedLabel")
+                : t("orders.reservations.notAllocatedLabel")}
```

Comment:
suggestion: can we wait with the rendering of this until we get the data because we get initial screen saying "Not allocated" and then it switches to "Allocated"